src/sbl_filing_api/entities/models/dto.py

```diff
@@ -71,6 +71,7 @@ class FilingDTO(BaseModel):
     contact_info: ContactInfoDTO | None = None
     confirmation_id: str | None = None
     signatures: List[UserActionDTO] = []
+    creator_id: int
```

Comment:
Same as the other user actions, we want to return 'creator' here of type UserActionDTO instead of just the id.